evalml/objectives/sensitivity_low_alert.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from .binary_classification_objective import BinaryClassificationObjective
+
+from evalml.utils.logger import get_logger
+
+logger = get_logger(__file__)
+
+
+class SLA(BinaryClassificationObjective):
```

Comment:
@skvorekn could we name this something like `SensitivityLowAlert`, just to give users a bit more context at first glance?

Reply:
Good call! Done!